aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/internal/UnsignedXrayClient.java

```diff
@@ -64,6 +77,17 @@ public boolean hasIgnoreMarker(AnnotatedMember m) {
                     }
                     return super.hasIgnoreMarker(m);
                 }
+
+                @Override
+                public PropertyName findNameForDeserialization(Annotated a) {
+                    if (a.getName().equals("hTTPMethod")) {
+                        return HTTP_METHOD;
+                    }
+                    if (a.getName().equals("uRLPath")) {
```

Comment:
Capitalization of the strings seems off

Reply:
Yeah these look weird but are the actual field names in the model class and is what jackson uses here.